lua/terrortown/menus/gamemode/equipment.lua

```diff
@@ -28,15 +28,14 @@ function CLGAMEMODEMENU:InitializeVirtualMenus()
 	for i = 1, #equipments do
 		local equipment = equipments[i]
 
-		-- Only keep ttt-equipments that are cached
-		if not equipment.ttt2_cached_material and not equipment.ttt2_cached_model then continue end
+		if not equipment.isEquipment then continue end
```

Comment:
This seems to filter out nearly all equipments that should be editable.

Reply:
I don't get what you mean. `.isEquipment` is true where `not equipment.ttt2_cached_material and not equipment.ttt2_cached_model` was true preiously. So nothing changed here except a name change

Reply:
Not completely. If it has a material thats not id OR a model thats not bugbait, it would have come through here.
But now it is only marked as equipment true if material is not id AND a model is not bugbait.